lib/pf/web/constants.pm

```diff
@@ -89,6 +89,7 @@ Readonly::Scalar our $URL_ADMIN_MANAGE_GUESTS   => '/guests/manage';
 
 Readonly::Scalar our $URL_GAMING_REGISTRATION   => '/gaming-registration';
 Readonly::Scalar our $URL_DEVICE_REGISTRATION   => '/device-registration';
+Readonly::Scalar our $URL_DEVICE_REGISTRATION_LOGOUT   => '/device-registration/logout';
```

Comment:
adjust the identation of the other variables so it matches the one you added. To my knowledge they are all equal so we should keep it this way